SalesOrderNewSection-1/Block/Adminhtml/Order/View/CustomContent.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace RightpointAssignment\SalesOrderNewSection\Block\Adminhtml\Order\View;
```

Comment:
SalesOrderNewSection-1/Block/Adminhtml/Order/View/CustomContent.php

File path and the name space not matching. Please correct it.
Can you rename the module as SalesOrderAdminCustamization?